server/service/integration_mdm_test.go

```diff
@@ -5662,6 +5661,14 @@ func (s *integrationMDMTestSuite) TestBootstrapPackage() {
 	s.DoJSON("DELETE", "/api/latest/fleet/mdm/bootstrap/0", nil, http.StatusNotFound, &deleteResp)
 }
 
+func (s *integrationMDMTestSuite) TestBootstrapStatusFilterOnlyMacOS() {
+	t := s.T()
+	pkg, err := os.ReadFile(filepath.Join("testdata", "bootstrap-packages", "signed.pkg"))
+	require.NoError(t, err)
+
+	s.uploadBootstrapPackage(&fleet.MDMAppleBootstrapPackage{Bytes: pkg, Name: "pkg.pkg", TeamID: 0}, http.StatusOK, "")
+}
```

Comment:
sanity check: should we remove this?

Reply:
Yep, was a start of a test that I didn't end up needing.